docs/control-panel/categories.md

```diff
@@ -32,6 +32,13 @@ Catagories have the following settings:
 - **Parent category** -- This drop-down list allows you to create a hierarchical relationship between categories. The list dynamically contains all existing categories for this category group. Selecting a parent category means that the new category will be a "child" of the parent in the hierarchy. The "None" option is available and will make the new category a "top level" category with no parent.
 - **Custom Fields** -- Any [custom category fields](#fields-tab) that exist for this category group will be included in the form.
 
+The categories can be saved by clicking "Save" button or pressing `Ctrl`+`s` (`cmd`+`s`) on keyboard. Upon saving, the category editing page remains open so you can make other changes.
+
+Extra saving options are available from dropdown menu under "Save": 
+- "Save & New" - opens new category form after saving
+- "Save & Close" - redirects to categories list after saving
+- "Clone to New Category" - saves the changes into new category. The current category is used as "template" and remains unchanged.
```

Comment:
@intoeetive With channel entries it creates a new entry... will the "new category" already be saved?  Or does the user have to save it?

Reply:
@TomJaeger it works the same way as entries. Is saves the moment you hit the button in Save dropdown